packages/viz/src/lib/Layer.ts

```diff
@@ -67,12 +70,10 @@ export class Layer {
    */
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   public async addTo(deckInstance: any) {
-    const currentDeckLayers = deckInstance.props.layers;
     const createdDeckGLLayer = await this._createDeckGLLayer();
 
-    deckInstance.setProps({
-      layers: [...currentDeckLayers, createdDeckGLLayer]
-    });
+    const map = deckInstance.getMapboxMap();
```

Comment:
I think this will break Google Maps functionality, we cannot change this like that. We need to come up with another solution.

Reply:
Agree, I talked about it in the slack channel: https://github.com/CartoDB/toolkit/pull/96/files#r422054110.

I'd go for the native solution. Something like this: https://deck.gl/#/examples/core-layers/geojson-layer-polygons